### DISCUSSION TITLE
Issue with Loading App Updates - "Could not load app discover section"

### DISCUSSION THREAD

**Discussion by vawaver** [nextcloud/all-in-one#5317] [Questions] [answered]
**Subject:** Issue with Loading App Updates - "Could not load app discover section" on Nextcloud AIO 9.5.1 (29.0.6)

Hello,

I am using Nextcloud AIO version 9.5.1 (29.0.6) running on a server with 15 GB RAM and a 12-core CPU, using Ubuntu server 24.04. When I try to access the Apps section, the list of app updates doesn't load, and I receive the error message: "Could not load app discover section."

To resolve this issue, I have already recreated all the Docker containers, but this did not help.

Does anyone have suggestions on how to fix this issue? 
Any guidance would be greatly appreciated.

![discover-section-issue](https://github.com/user-attachments/assets/9b980ecd-c9e5-472c-9e35-e428374215ae)

**Comment by szaimen** [comment, marked as answer]
Hi, see https://help.nextcloud.com/t/discover-section-of-the-app-store-not-working/204688

**Comment by vawaver** [comment]
Hi,

I am very glad for the information. I will wait for the Nextcloud client update.

I hope the new version will address the issues with syncing large files (over 2 GB). Currently, my server shuts down during the finalization of the upload process for such large files.

For your reference, my server is running on an NVMe disk with 15 GB of RAM and a 12-core CPU. 
Maybe it is another issue.

**Reply by szaimen** [reply]
Yes, this is a different issue. See https://github.com/nextcloud/all-in-one/discussions/5318

**Reply by vawaver** [reply]
Can you check URL for this second issue? 
Because #5318 is this topic.
